.idea/codeStyleSettings.xml

```diff
@@ -16,7 +16,7 @@
             <package name="" withSubpackages="true" static="false" />
           </value>
         </option>
-        <option name="ENABLE_JAVADOC_FORMATTING" value="false" />
+        <option name="RIGHT_MARGIN" value="80" />
```

Comment:
80 is generally considered too small for java. 80 + 3-char indent more so. (Google uses 100+2).